frontend/uddhar-frontend/src/shared/components/DashboardNavbar.jsx

```
@@ -22,17 +27,15 @@ const DashboardNavbar = ({ children }) => {
+            <div onClick={()=>{setUserMenuOpen(prev => !prev)}}>
+              <Avatar className="w-10 h-10" {...config} />
+              <span>{user.email.length > 5 ? `${user.email.slice(0, 5)}...` : ""}</span>
+            </div>
             {userMenuOpen && (
               <div className="absolute right-0 mt-2 w-40 bg-white border rounded-lg cursor-pointer">
-                <p className="p-2 text-gray-700 hover:bg-blue-200">Profile</p>
-                <p className="p-2 text-gray-700 hover:bg-blue-200">Settings</p>
-                <p className="p-2 text-gray-700 hover:bg-blue-200">Logout</p>
+                <nav className="p-2 text-gray-700 hover:bg-blue-200">Profile</nav>
+                <nav className="p-2 text-gray-700 hover:bg-blue-200">Settings</nav>
+                <nav className="p-2 text-gray-700 hover:bg-blue-200" onClick={logout}>Logout</nav>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)